plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -2183,4 +2183,27 @@ public void testAggregationPushdownWithArrays()
                         "(56, VARCHAR 'string_8400', BIGINT '1')," +
                         "(1000, VARCHAR 'string1_8401', BIGINT '1')");
     }
+
+    @Test
+    public void testHavingClause()
+    {
+        assertThat(query("SELECT stringcol, \"sum(longcol)\"" +
+                "  FROM \"SELECT stringcol, SUM(longcol) FROM hybrid" +
+                "  GROUP BY stringcol HAVING SUM(longcol) > 1\""))
+                .matches("  VALUES (VARCHAR 'string_15', DOUBLE '3.0')," +
```

Comment:
```suggestion
                .matches("VALUES (VARCHAR 'string_15', DOUBLE '3.0')," +
```

Reply:
Had to change the table: there is a bug in the secure pinot test setup where the server gets access denied but pinot 0.9.3 doesn't have a way to pass basic auth. I will fix at it in another pr to correct the auth setup.